.github/renovate.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": [
+    "config:js-app",
+    ":gitSignOff",
+    "npm:unpublishSafe",
+    "group:linters",
+    "helpers:pinGitHubActionDigests"
```

Comment:
I think this will pin to the latest commit, not the latest release. You need to annotate your GH actions with `# <tag>` comment like this for desired version:


```yaml
        uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3
```

You can leave this be and check if it actually happens once renovate opens a PR for it. Just be aware that the sha/commit renovate is proposing you to use, may not be part of a release yet.